tests/virtctl/create_vm.go

```diff
@@ -59,6 +61,67 @@ var _ = Describe("[sig-compute][virtctl]create vm", func() {
 			Expect(*vm.Spec.RunStrategy).To(Equal(v1.RunStrategyAlways))
 		})
 
+		It("Example with volume-import flag and PVC type", func() {
+			const runStrategy = v1.RunStrategyManual
+
+			pvc := libstorage.CreateFSPVC("vm-pvc-"+rand.String(5), util.NamespaceTestDefault, size, nil)
+
+			out, err := clientcmd.NewRepeatableVirtctlCommandWithOut(create, VM,
+				setFlag(RunStrategyFlag, string(runStrategy)),
+				setFlag(VolumeImportFlag, fmt.Sprintf("type:pvc,size:%s,name:%s,namespace:%s", size, pvc.Name, util.NamespaceTestDefault)),
+			)()
+			Expect(err).ToNot(HaveOccurred())
+			vm, err := virtClient.VirtualMachine(util.NamespaceTestDefault).Create(context.Background(), unmarshalVM(out))
+			Expect(err).ToNot(HaveOccurred())
+
+			Expect(vm.Spec.Template.Spec.Volumes).To(HaveLen(1))
+			Expect(vm.Spec.RunStrategy).ToNot(BeNil())
+			Expect(*vm.Spec.RunStrategy).To(Equal(runStrategy))
+			Expect(vm.Spec.DataVolumeTemplates[0].Spec.Source.PVC).ToNot(BeNil())
+			Expect(vm.Spec.DataVolumeTemplates[0].Spec.Source.PVC.Name).To(Equal(pvc.Name))
+			Expect(vm.Spec.DataVolumeTemplates[0].Spec.Source.PVC.Namespace).To(Equal(util.NamespaceTestDefault))
+			Expect(vm.Spec.DataVolumeTemplates[0].Spec.Storage.Resources.Requests[k8sv1.ResourceStorage]).To(Equal(resource.MustParse(size)))
+		})
+
+		It("Example with volume-import flag and Registry type", func() {
+			const runStrategy = v1.RunStrategyManual
+
+			cdSource := cd.ContainerDiskFor(cd.ContainerDiskCirros)
+
+			out, err := clientcmd.NewRepeatableVirtctlCommandWithOut(create, VM,
+				setFlag(RunStrategyFlag, string(runStrategy)),
+				setFlag(VolumeImportFlag, fmt.Sprintf("type:registry,size:%s,url:%s", size, cdSource)),
```

Comment:
Does it actually work without the `docker://` prefix? You could wait for the VM to be ready.